openfl/federated/data/loader_keras.py

```diff
@@ -99,7 +107,9 @@ def _batch_generator(X, y, idxs, batch_size, num_batches):
             b = a + batch_size
             yield X[idxs[a:b]], y[idxs[a:b]]
 
-    def _get_batch_generator(self, X, y, batch_size, num_batches=None):
+    def _get_batch_generator(self, X: np.ndarray, y: np.ndarray,
+                             batch_size: Optional[int], num_batches: Optional[int] = None
```

Comment:
Here `batch_size` is `Optional[int]`, but does not have the default value. This might be confusing.